lib/nhsuk-prototype-kit.js

```diff
@@ -30,12 +31,35 @@ NHSPrototypeKit.middleware = middleware
 // This does all setup for both the Express app
 // and the Nunjucks environment
 NHSPrototypeKit.init = function (options) {
+  options.express = express()
+
   // Set all the Express settings
   expressSettings.setAll(options.express)
 
+  let nunjucksViews = []
+
+  if (options.appViews) {
+    nunjucksViews.push(options.appViews)
+  }
+
   // Add the prototype kit view path to Nunjucks environment
   const prototypeKitViewsPath = join(import.meta.dirname, 'views')
-  options.nunjucks.loaders[0].searchPaths.push(prototypeKitViewsPath)
+  nunjucksViews.push(prototypeKitViewsPath)
+
+  // NHS.UK frontend package path
+  const nhsukFrontendPath = fileURLToPath(
```

Comment:
Is it time to share this with other files from somewhere?

I like the `nhsuk` prefix but aren't all the others like this? Change them all if you like

```suggestion
  const frontendPath = fileURLToPath(
```

Reply:
I thought they all had the same value, the directory where package.json is?